### PR TITLE
i3status-rust: update to 0.32.2.

### PR DESCRIPTION
**Pull request by pablomancera**
#### Testing the changes
- I tested the changes in this PR: **YES**

#### Local build testing
- I built this PR locally for my native architecture, x86_64
- I built this PR locally for these architectures (if supported. mark crossbuilds):
  - aarch64 (crossbuild)
  - aarch64 (local)
  - armv6l (crossbuild)

Supersedes #44899

### CODE REVIEW

**Comment by Duncaen** [comment]
Why is generating man pages a build option?

File: srcpkgs/i3status-rust/template [hunk 1]
```diff
@@ -1,23 +1,41 @@
...
+	if [ "$build_option_docs" ]; then
+		env -i bash -c "source /etc/profile && cargo xtask generate-manpage"
+	fi
+}
 
 post_install() {
 	vmkdir usr/share/i3status-rust
-	vcopy ${wrksrc}/files/icons usr/share/i3status-rust/
-	vcopy ${wrksrc}/files/themes usr/share/i3status-rust/
+	vcopy ${wrksrc}/files/* usr/share/i3status-rust/
```
Comment (classabbyamp) [inline srcpkgs/i3status-rust/template:36]: wrksrc is the working directory, it doesn't need to be specified

**Comment by pablomancera** [comment]
> Why is generating man pages a build option?

Because it uses pandoc to generate them, so some architectures won't be able to build the package if they also have to generate the man pages.

**Comment by Duncaen** [comment]
But its enabled by default effectively removing the package for the architectures we cross compile pacakges for.
This should be set if cross compiling, but not sure if a build option is really required here, you could just add pandoc to hostmakedepends if not cross compiling.

**Comment by pablomancera** [comment]
The idea of the line

`env -i bash -c "source /etc/profile && cargo xtask generate-manpage"`

is that the void build servers (which I think are x86_64) generates the man page using the packages native to their architecture and bundle them in the package for every other architecture. It's the solution I came up with after the review I got at #44899 so every architecture can get the man pages. I'm not entirely sure it's the best though, but it worked in my tests (local building for x86_64 and cross building for aarch64). If there is a better way of doing this I would like to hear it.

On the other hand, if anyone wants to build the package on an architecture limited by the pandoc package, they can disable the docs build option and it'll be fine.

That's the reason I added a docs build option.

**Pull request closed without merging**